mart/configs/model/detection/preprocessor_multi_modal.yaml

```diff
@@ -0,0 +1,12 @@
+# @package model.modules.preprocessor
+_target_: mart.transforms.TupleTransforms
+transforms:
+  _target_: torchvision.transforms.Compose
+  transforms:
+    - _target_: mart.transforms.GetItems
+      keys: ${datamodule.test_dataset.modalities}
```

Comment:
I don't think this is correct. These keys need to be specified independently because I could load the images in [depth, rgb] order but require they be order as [rgb, depth] for the model.

Reply:
Would also be nice if this could use the yaml file below (preprocessor_single_modal.yaml).

Reply:
Now it use the other yaml file.

I think it's convenient to use interpolation by default. We can always change the keys in `experiment.yaml` or in the command line if we encounter that rare situation.

Reply:
But isn't there a silent bug if I switch the datamodule modality from [rgb, depth] to [depth, rgb]?

Reply:
Sure...